src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -287,6 +292,31 @@ export const gravityStitchingEnvironment = (
         },
       },
       MarketingCollection: {
+        thumbnailImage: {
+          fragment: gql`
+          ... on MarketingCollection {
+            image_url: thumbnail
+          }
+          `,
+          resolve: async ({ image_url }, args, context, info) => {
+            if (image_url) {
+              context.imageData = {
+                image_url,
+              }
+            } else {
+              context.imageData = null
+            }
+
+            return info.mergeInfo.delegateToSchema({
+              args,
```

Comment:
**question**: Do these args contain any data? Or is the image type fulfilled entirely by the context manipulation above?

Reply:
removed args